src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,142 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegisterException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final User FIRST_USER = new User("login1", "password", 18);
+    private static final User SECOND_USER = new User("login113213", "password32 123321", 45);
+    private static final User NULL_LOGIN_USER = new User(null, "password", 18);
+    private static final User NULL_PASSWORD_USER = new User("login1", null, 18);
+    private static final User NULL_AGE_USER = new User("login1", "password", null);
+    private static final User UNDER_MIN_LOGIN_USER = new User("log", "password", 18);
+    private static final User UNDER_MIN_PASSWORD_USER = new User("login1", "pass", 18);
+    private static final User UNDER_MIN_AGE_USER = new User("login1", "password", 10);
+    private static final User UNDER_MIN_VALUES_USER = new User("login", "123", 16);
+    private static final User ZERO_VALUES_USER = new User("", "", 0);
+    private static final User NULL_VALUES_USER = new User(null, null, null);
+    private static final int EXPECTED_SIZE = 2;
+    private static RegistrationService registrationService;
+    private static StorageDao storageDao;
+    private Storage storage;
+
+    @BeforeAll
+    public static void beforeAll() {
+        storageDao = new StorageDaoImpl();
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    public void setUp() {
+        storage = new Storage();
+    }
+
+    @Test
+    public void register_nullAge_notOk() {
+        assertThrows(Exception.class, () -> {

```

Comment:
Use your custom exception
```suggestion
        assertThrows(RegisterException.class, () -> {
```